### ISSUE TITLE
Keyboard Accessibility Issue with Table Headers When Sorting is Disabled

### ISSUE THREAD

**Issue by pankajviv** [TobiasBg/TablePress#222]
Hi TablePress Team,

We have identified an accessibility issue with table headers when sorting is disabled in TablePress.

You can see this on the following page: https://s1famfirst.wpengine.com/find-a-program/

Even when sorting is turned off in the table settings, the table header elements still include DataTables-related interactive markup, such as:

* dt-orderable-none class on table header elements
* dt-column-header wrapper inside each header
* In some cases, tabindex="0" is applied, making headers focusable

As a result, the headers appear interactive and focusable to assistive technologies, but they do not provide any keyboard functionality.

This is being flagged as a WCAG 2.1 (Success Criterion 2.1.1 – Keyboard) accessibility violation:
"Function cannot be performed by keyboard alone"

Current DOM output (example):

```html
<th class="column-2 dt-orderable-none" tabindex="0">
    <div class="dt-column-header">Column Title</div>
</th>
```

Expected behavior:

When sorting is disabled, headers should not be interactive or keyboard-focusable.

Option 1 (minimum fix):

```html
<th class="column-2 dt-orderable-none" tabindex="-1">
    <div class="dt-column-header">Column Title</div>
</th>
```

Option 2 (preferred):

```html
<th class="column-2">Column Title</th>
```

Could you please confirm:

* Is there a way to fully suppress DataTables interactive markup when sorting is disabled?
* Or can this be addressed in a future update for better accessibility compliance?

Since this markup is generated internally by Table Press/Data Tables, we do not have direct control over it.

Thanks!
